test/unit/points/test-calculate-points-for-workload.js

```diff
@@ -0,0 +1,30 @@
+const expect = require('chai').expect
+const calculateTierWorkloadPoints = require('../../../app/points/calculate-points-for-workload')
+const pointsHelper = require('../../helpers/points-helper')
+const Locations = require('../../../app/staging/constants/locations')
+
+describe('points/calculate-community-tier-workload-points', function () {
```

Comment:
What about misuse cases such as incorrect parameter types? Because we're working with a untyped language we need to ensure that we're catching these error scenarios in the code.